packages/google_sign_in/lib/google_sign_in.dart

```diff
@@ -128,7 +128,7 @@ class GoogleSignIn {
       );
     }
     await _initialization;
-    Map<String, dynamic> response = await _channel.invokeMethod(method);
+    Map<String, dynamic> response = await _channel.invokeMethod(method,null);
     _currentUser = response != null ? new GoogleSignInAccount._(this, response) : null;
```

Comment:
@goderbauer , @ajonp 

When testing locally the returned `response` for me is an empty `Map`, so on disconnect it tries to instantiate an account and fails on assertions in the constructor.